theme/static/js/create-resource.js

```diff
@@ -21,95 +143,132 @@ $(document).ready(function() {
             $("#file-move-warning").hide();
         }
     });
-    $('#resource-type').on('change', function(){
-        $('#select-file').value = '';
-        $('#select-file').attr('value', '');
-        $('#irods-sel-file').text("No file selected.");
```

Comment:
Wait, I just noticed I updated with develop branch instead of the release target by mistake. Will take me a while to revert :(